src/room/GroupCallView.tsx

```diff
@@ -495,9 +495,7 @@ export const GroupCallView: FC<Props> = ({
     }
   } else if (left && widget !== null) {
     // Left in widget mode:
-    if (!returnToLobby) {
-      body = null;
-    }
+    body = returnToLobby ? lobbyView : null;
   } else if (preload || skipLobby) {
```

Comment:
Could the whole block be changed to:
```
} else if (left && widget && !returnToLobby) {
    // Left in widget mode:
    body = null;
} else ...
  ```
  Then the final `else` block would set `body` to `lobbyView`

Reply:
The `preload || skipLobby` case seems to complicate this, enough that I wasn't sure whether an `else if (left && widget && !returnToLobby)` would really do the right thing. So I stuck to structuring this `left && widget !== null` branch in the same way as the `left && widget === null` branch, which I think brings its own kind of clarity.